packages/maskbook/src/plugins/Wallet/database/Wallet.db.ts

```diff
@@ -49,50 +51,19 @@ export const createWalletDBAccess = createDBAccess(() => {
                 // @ts-ignore
                 db.deleteObjectStore('RedPacket')
             }
-            /**
-             * Use checksummed address in DB
-             */
-            async function v3_v4() {
-                const t = createTransaction(db, 'readwrite')('Wallet', 'ERC20Token')
-                const wallets = t.objectStore('Wallet')
-                const tokens = t.objectStore('ERC20Token')
-                for await (const wallet of wallets) {
-                    // update address
-                    wallet.value.address = formatChecksumAddress(wallet.value.address)
 
-                    // update token list sets
-                    ;[wallet.value.erc20_token_blacklist, wallet.value.erc20_token_whitelist].forEach((set) => {
-                        const values = Array.from(set.values())
-                        set.clear()
-                        values.forEach((value) => set.add(formatChecksumAddress(value)))
-                    })
-                    await wallet.update(wallet.value)
-                }
-                for await (const token of tokens) {
-                    token.value.address = formatChecksumAddress(token.value.address)
-                    await token.update(token.value)
-                }
-            }
             /**
-             * Fix providerType does not exist in legacy wallet
+             * Store ERC721Token records in DB
              */
-            async function v4_v5() {
-                const t = createTransaction(db, 'readwrite')('Wallet')
-                const wallets = t.objectStore('Wallet')
-                for await (const wallet of wallets) {
-                    const wallet_ = wallet as any
-                    if (wallet_.value.provider) continue
-                    if (wallet_.value.type === 'managed') wallet_.value.provider = ProviderType.Maskbook
-                    else if (wallet_.value.type === 'exotic') wallet_.value.provider = ProviderType.MetaMask
-                    await wallet.update(wallet_.value)
-                }
+            function v5_v6() {
+                db.createObjectStore('ERC721Token', { keyPath: path<keyof ERC721TokenRecordInDatabase>('record_id') })
+                db.createObjectStore('ERC1155Token', { keyPath: path<keyof ERC1155TokenRecordInDatabase>('record_id') })
             }
 
             if (oldVersion < 1) v0_v1()
             if (oldVersion < 2) v1_v2()
-            if (oldVersion < 3) await v2_v3()
-            if (oldVersion < 4) await v3_v4()
-            if (oldVersion < 5) await v4_v5()
+            if (oldVersion < 3) v2_v3()
```

Comment:
Why v3 to v5 migration removed?

Reply:
v4 and v5 are broken not working actually.